style.css

```diff
@@ -0,0 +1,632 @@
+  align-items: center;
+  background-color: var(--primary-color);
+  color: var(--quaternary-color);
+  border-bottom: 2px solid black;
+  z-index: 5;
+}
+#logo {
+  font-size: 1.7rem;
+  font-weight: 700;
+}
+nav > ul > li > a {
+  color: black;
+  font-weight: 500;
+}
+nav > ul {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20%;
+}
+#mobile-nav {
+  display: none;
+}
+/* Main */
+main {
+  margin-top: 10vh;
+}
+#hero-section {
+  min-height: 90vh;
+  background-image: url("./assets/images/sulpture01.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  display: grid;
+  grid-template-columns: 1fr 4fr 1fr;
+  align-items: center;
+  place-items: center;
+}
+#hero-section-aside-1,
+#hero-section-aside-2 {
+  min-height: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+#hero-section-aside-1 > h2 {
+  font-size: 1.9rem;
+  font-weight: 700;
+  color: var(--tertiary-color);
+}
+#hero-section-aside-1 > img {
+  height: 20vh;
+}
+#hero-section-aside-2 > img {
+  height: 30vh;
+}
+#hero-section-aside-2 > div > ul > li {
+  border-bottom: 2px solid var(--tertiary-color);
+  padding-top: 1rem;
+  padding-bottom: 0.4rem;
+}
+#hero-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+#hero-section-header {
+  font-size: 5rem;
+  color: var(--quaternary-color);
+  padding: 2rem 4rem;
+
+}
+#cta-btn {
+  padding: 0.8rem 1.5rem;
+  background-color: var(--tertiary-color);
+  color: var(--primary-color);
+  font-size: 1.2rem;
+  border: none;
+  border-radius: 10px;
+  cursor: pointer;
+}
+#hero-section-tags {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.2rem;
+  margin-top: 1rem;
+}
+.hero-text-style {
+  text-align: end;
+}
+
+#hero-section-tags > span {
+  font-size: 0.9rem;
+  font-weight: 600;
+  border: black solid 1px;
+  border-radius: 20px;
+  padding: 0.2rem 0.4rem;
+  color: var(--tertiary-color);
+}
+#hero-section-tags > span:hover {
+  background-color: var(--tertiary-color);
+  color: var(--primary-color);
+}
+
+#services-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+  margin-top: 2rem;
+}
+.service-card {
+  background-color: white;
+  text-align: center;
+  border: 2px solid var(--quaternary-color);
+  border-radius: 10px;
+  height: 100%;
+  padding: 1rem 2rem;
+}
+.service-card > svg {
+  height: 2.5rem;
+}
+.service-card > h3 {
+  font-size: 1rem;
+  font-weight: 700;
+  color: var(--quaternary-color);
+  margin: 0.8rem 0;
+}
+.service-card > p {
+  font-size: 0.9rem;
+  margin-bottom: 1rem;
+}
+#pricing-table {
+  width: 100%;
+  border-collapse: collapse;
+  margin-top: 20px;
+}
+
+#pricing-table th,
+#pricing-table td {
+  text-align: left;
+  padding: 16px;
+  font-size: 1rem;
+}
+
+#pricing-table th {
```

Comment:
- avoid using same css property multiple times 
- pricing-table td and th use one time not multiple times